index.html

```diff
@@ -1,13 +1,16 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Color Compass - Chingu Voyage 49 Team 16</title>
-    <script defer type="module" src="/src/main.jsx"></script>
+    <title>Vite + React</title>
```

Comment:
we are removing the title and reverting to default.

Reply:
OK!